rental.rb

```diff
@@ -0,0 +1,12 @@
+class Rental
+  attr_accessor :date
+  attr_reader :book, :person
```

Comment:
- [ ] Since a rental belongs to a person :+1: , a person should have many rentals. Now you'll kindly show that a person has many rentals by a method in the **Person** class that serves this purpose. [ReadMore..](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_relationships_by_examples.md#many-to-many-relationship)